Dockerfile

```
@@ -1,5 +1,29 @@
-FROM clojure:tools-deps-alpine
-LABEL Name=clojure-representer Version=0.0.1
-COPY . /opt/representer
-WORKDIR /opt/representerer
+FROM babashka/babashka:0.10.163-alpine as builder
+
+# Install SSL ca certificates
+RUN apk update && apk add ca-certificates
+
+# Create appuser
+RUN adduser -D -g '' appuser
+
+# get the source code
+WORKDIR /clojure-representer
+COPY clojure_representer.clj .
+COPY bin/run.sh bin/run.sh
+
+# Get static analyzer image as build stage
+FROM cljkondo/clj-kondo:2021.01.20-alpine as clj-kondo
+
+# Build a minimal and secured container
+FROM babashka/babashka:0.10.163-alpine
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /etc/passwd /etc/passwd
+COPY --from=builder /clojure-representer/clojure_representer.clj /opt/representer/clojure_representer.clj
+COPY --from=builder /clojure-representer/bin/run.sh /opt/representer/bin/run.sh
```

I think you could also do this.

```suggestion
WORKDIR /opt/test-runner
COPY --from=build /clojure-representer/ .
```

Which means that you could move the `COPY` command after the existing `WORKDIR /opt/representer` line.